### DISCUSSION TITLE
Add Documentation (README.MD) to Functions and Tools

### DISCUSSION THREAD

**Discussion by hchasens** [open-webui/open-webui#4221] [Ideas]
The open-webui website allows users to find tools and functions to download, but there's no space for documentation or comments. This is frustrating as it doesn't allow users to ask questions surrounding a tool or function, nor for the author to provide additional documentation. While there is an opportunity for inline comments (via Sphinx), I think a dedicated description would be extremely helpful for users. 

While many functions are short enough to where documentation isn't necessary, for a function such as (https://openwebui.com/f/atgehrhardt/artifacts) documentation and comment would provide users with much-needed context and aid.

**Comment by justinh-rahb** [comment]
It's on the to-do list to be addressed most immediately.

**Reply by hchasens** [reply]
yay! Thank you!
